Gems/PhysX/Code/Source/Scene/PhysXSceneSimulationFilterCallback.cpp

```diff
@@ -55,7 +55,10 @@ namespace PhysX
     size_t SceneSimulationFilterCallback::CollisionPairHasher::operator()(const CollisionActorPair& collisionPair) const
     {
         size_t hash{ 0 };
-        AZStd::hash_combine(hash, collisionPair.m_actorA, collisionPair.m_actorB);
+        // Order elements so {1,2} and {2,1} would generate the same hash
+        const physx::PxActor* smallerVal = AZStd::min(collisionPair.m_actorA, collisionPair.m_actorB);
+        const physx::PxActor* biggerVal = AZStd::max(collisionPair.m_actorA, collisionPair.m_actorB);
```

Comment:
nit: You should be able to use AZStd::minmax that returns a std::pair 🙂 - https://en.cppreference.com/w/cpp/algorithm/minmax (if it's not in `AZStd` for some reason could you please add it 😝 (I'll send you an accolade to make up for the compile time 😉)

Reply:
`AZStd::minmax` was added years ago in [algorithm.h](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/std/algorithm.h#L43-L47)